spec/system/admin/configuration/terms_of_service_files_spec.rb

```diff
@@ -14,33 +14,39 @@
       visit spree.edit_admin_general_settings_path
       click_link "Terms of Service"
 
-      expect(page).to have_content "No terms of services have been uploaded yet."
-      expect(page).to have_content "your old Terms of service"
-      expect(page).to have_link "Terms of service", href: "/Terms-of-service.pdf"
+      expect(page).to have_content I18n.t("admin.terms_of_service_files.show.no_files")
+      expect(page).to have_content(
+        I18n.t("admin.terms_of_service_files.show.using_default_terms_html",
+               tos_link: I18n.t(:terms_of_service))
+      )
+      expect(page).to have_link I18n.t(:terms_of_service), href: "/Terms-of-service.pdf"
```

Comment:
We are still discussing this but I think that @filipefurtad0 and I agree that it would be better to revert this change and use plain text instead of I18n.t calls. It's actually more reliable. It means that we are checking that your introduction of translations doesn't break the previous English version.

Reply:
So, do you mean to change every I18n.t calls of this file?

> We are still discussing this but I think that @filipefurtad0 and I agree that it would be better to revert this change and use plain text instead of I18n.t calls. It's actually more reliable. It means that we are checking that your introduction of translations doesn't break the previous English version.

Reply:
> mean to change every I18n.t calls of this file
Yes, that would be perfect @aintluks :+1: